core/components/button/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Loader2 as Spinner } from 'lucide-react';
```

Comment:
Is there a reason we added a new component instead of modifying our existing button?

Reply:
No reason, at first I was trying to include more logic but since it's nod needed I can delete this.

Reply:
Moved code to `ui/button`.